CHANGELOG.md

```diff
@@ -1,117 +1,138 @@
+## 12.0.0
+
+_Released `2022-03-23`_
+
+We're breaking slightly from our pattern of a single yearly major release. The `12.0.0` release contains some backwards-incompatible changes to how middleware and auth refreshes work. For the most part, you'll be able to upgrade to this version safely, but as always, it's worth re-running unit tests (especially those related to authentication).
+
+In the coming months, we'll follow up with a `13.0.0` release that will bump the Node.js runtime and dependencies (ending support for Node.js 12 as it reaches End of Life). We're hoping that by separating these releases, the upgrade process will be easier for developers (only worrying about the public API or the runtime, but not both).
+
+### cli
+
+- None!
+
+### core
+
+- :exclamation: calling `response.throwForStatus()` now **always** throws an error if the response status code is `>= 400`. Previously it was a no-op when `response.skipThrowForStatus` was `true`. Now, that flag only controls whether Zapier's built-in middleware calls `throwForStatus()`. This only affects you if you set `skipThrowForStatus` and always call `.throwForStatus()`, expecting it not to error. ([#511](https://github.com/zapier/zapier-platform/pull/511))
+- :exclamation: re-add the built-in auto-refresh middleware for `oauth2` and `session` auths. This runs _before_ your declared `afterResponse`, so you no longer have to account for stale credentials in your middleware (unless you want to). See [the README](https://github.com/zapier/zapier-platform/blob/master/packages/cli/README.md#using-http-middleware) for more info. ([#512](https://github.com/zapier/zapier-platform/pull/512), [#517](https://github.com/zapier/zapier-platform/pull/517))
+
+### schema
+
+- :nail_care: add app-wide skipThrowForStatus flag. This is helpful for backwards compatibility when migrating from `9.x` to `12.x`, but probably won't be relevant for most developers. ([#511](https://github.com/zapier/zapier-platform/pull/511))
+
 ## 11.3.3
 
 _released `2022-03-21`_
 
 ### cli
 
-* :scroll: Update [README.md](README.md) and add [CONTRIBUTING.md](CONTRIBUTING.md) ([#506](https://github.com/zapier/zapier-platform/pull/506))
+- :scroll: Update [README.md](README.md) and add [CONTRIBUTING.md](CONTRIBUTING.md) ([#506](https://github.com/zapier/zapier-platform/pull/506))
 
 ### core
 
-* :bug: Fix another hanging issue where an action could be invoked multiple times ([#513](https://github.com/zapier/zapier-platform/pull/513))
+- :bug: Fix another hanging issue where an action could be invoked multiple times ([#513](https://github.com/zapier/zapier-platform/pull/513))
 
 ### schema
 
-* None!
+- None!
```

Comment:
prettier converted all the bullets from `*` into `-`. I'm not sure if you run prettier on markdown, but I figured it's nice to be consistent here (and my `prettier` gets run on-save)